### PR TITLE
Tammy branch

### PR DESCRIPTION
**Pull request by tlim045**
Add basic html and css to the webapp.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -6,13 +6,109 @@
...
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+
+  <body style="padding-top:18%;" id = "index" >
```
Comment (wennyyustalim) [inline portfolio/src/main/webapp/index.html:11]: would prefer moving this to .css file :)

Comment (wennyyustalim) [reply portfolio/src/main/webapp/index.html:11]: nit: remove whitespace for consistency `..." id="index">`

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,230 @@
...
+    font-family: Nunito,sans-serif;
+    line-height: 1.5;
+    font-weight: 600;
+}
+
+a:hover {
+    color:#FBBC05 ;
+  }
+
+
```
Comment (wennyyustalim) [inline portfolio/src/main/webapp/style.css:22]: Nit: Remove the 2 extra whitespace if not needed

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (tlim045) [reply portfolio/src/main/webapp/index.html]: Okay! moved to css file and cleaned up the whitespaces

**Pull request merged**